### DISCUSSION TITLE
Small png no images are displayed

### DISCUSSION THREAD

**Discussion by gilipsy** [oleeskild/obsidian-digital-garden#334] [Q&A]
Small png images, approximately 30-50-70 kb are not displayed on the website

**Comment by dayne** [comment]
Are you using Vercel as your hosting provider?  I was just noticing issues with small png files not being displayed on my garden too and am in process of debugging it.   

I can view source and see the HTML code (good) and the relative URL for the image.  I can go to the website + the relative URL and the image is presented to me.  It just is not rendering inline on the page and I'm getting only the alt text. 

FWIW I'm using SynthWave as the theme.  About to test changing themes to see if that triggers a different behavior.

**Reply by dayne** [reply]
Of course the solution in this case was to make sure I was running latest version of the template.  

Obsidian -> Settings -> Community Plugins -> Manage site template -> Create PR
wait a bit
Merge the PR on GitHub 
wait a bit

Images fixed on website.
